src/utils/docs.jl

```diff
@@ -111,9 +111,23 @@ function open_doc()
     end
 end
 
+function start_doc_preview_server(;open_browser::Bool = true, port::Int = 8000)
+  build_dir = normpath(Oscar.oscardir, "docs", "build")
+  cmd = """
+        using Pkg;
+        Pkg.activate(temp = true);
+        Pkg.add("LiveServer");
+        using LiveServer;
+        LiveServer.serve(dir = "$build_dir", launch_browser = $open_browser, port = $port);
+        """
+  live_server_process = run(`julia -e $cmd`, wait = false)
```

Comment:
```suggestion
  live_server_process = run(`$(Base.julia_cmd()) -e $cmd`, wait = false)
```
There might not be a julia binary in the path? (this might also avoid the forking by juliaup)